client/src/components/FormAction/FormAction.scss

```diff
@@ -306,7 +306,7 @@ button.close {
 }
 
 .actions-warning {
-  color: darken($state-modified-bg, 15);
+  color: #856404; // in bootstrap: theme-color-level(map-get($theme-colors, "yellow"), 6)
```

Comment:
Are we able to use a variable, out of interest?

Reply:
I had opted out of using a variable for now, if we're finding ourselves using more of bootstrap's mixin'd theme colours we can setup a block in the _variables.scss file